GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -405,11 +405,23 @@ protected void ValidatePathParameter(string path)
         {
             if (!repo.CommitAndRootTreeExists(commitId))
             {
-                if (!gitObjects.TryDownloadCommit(commitId))
-                {
-                    error = "Could not download commit " + commitId + " from: " + Uri.EscapeUriString(objectRequestor.CacheServer.ObjectsEndpointUrl);
-                    return false;
-                }
+                return this.ForceTryDownloadCommit(commitId, objectRequestor, gitObjects, out error);
+            }
+
+            error = null;
+            return true;
+        }
+
+        protected bool ForceTryDownloadCommit(

```

Comment:
I don't see anything being "forced" here. This is really referring to the _other_ method, which first checks the cache. I would suggest just adding a `skipLocalCacheCheck` param to the existing method instead.

Reply:
:+1: